### ISSUE TITLE
Uncaught TypeError: Cannot read property 'dispose' of null

### ISSUE THREAD

**Issue by batjko** [olmokramer/atom-move-status-items#2]
1. Install package (v0.0.5)
2. Reload Atom
3. Try to click and drag individual elements on status bar in any direction.

**Atom Version**: 1.0.0
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [move-status-items](https://github.com/olmokramer/atom-move-status-items) package, v0.0.5

### Stack Trace

Uncaught TypeError: Cannot read property 'dispose' of null

```
At C:\Users\free\.atom\packages\move-status-items\lib\move-status-items.js:80

TypeError: Cannot read property 'dispose' of null
    at dragEnd (C:\Users\free\.atom\packages\move-status-items\lib\move-status-items.js:80:17)
    at HTMLBodyElement.<anonymous> (C:\Users\free\.atom\packages\move-status-items\lib\move-status-items.js:61:12)
    at HTMLBodyElement.jQuery.event.dispatch (C:\Users\free\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
    at HTMLBodyElement.elemData.handle (C:\Users\free\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "minimap",
      "enhanced-package-list",
      "themed-settings",
      "coffee-navigator",
      "typewriter-sounds",
      "mechanical-keyboard",
      "atom-terminal-panel",
      "atom-ungit"
    ]
  },
  "move-status-items": {
    "left": [
      "div[is=\"status-bar-file\"]",
      "div[is=\"status-bar-cursor\"]",
      "div[is=\"status-bar-selection\"]",
      "div[is=\"space-pen-div\"]",
      "div[is=\"space-pen-div\"]",
      "div[is=\"space-pen-div\"]"
    ],
    "right": [
      "div[is=\"space-pen-div\"]",
      "encoding-selector-status",
      "grammar-selector-status",
      "div.git-view[is=\"status-bar-git\"]",
      "div[is=\"space-pen-div\"]"
    ]
  }
}
```

### Installed Packages

```coffee
# User
file-icons, v1.5.8
git-log, v0.4.1
language-avs, v1.0.2
language-ejs, v0.2.0
linter-coffeelint, v0.3.2
move-status-items, v0.0.5
pigments, v0.8.2
script, vundefined
symbols-tree-view, v0.9.3
terminal-panel, v1.14.1
travis-ci-status, v0.15.1

# Dev
status-bar, v0.74.0
```

**Comment by olmokramer** [comment]
Thanks for the report. I just published v0.0.6, which fixes some things. Does that version work?

**Comment by batjko** [comment]
This indeed now doesn't happen anymore.
I'm afraid there's now new issues, but I'll open that separately.
